README.md

```diff
@@ -77,8 +77,9 @@ sessions expires automatically after 30 minutes of inactivity.
 ### Configuring login items in 1Password
 
 In order to show only relevant login items and to maintain compatibility with
-[sudolikeaboss](https://github.com/ravenac95/sudolikeaboss), its required to set the value of the
-`website` field for each login item with the value of `sudolikeaboss://local`.
+[sudolikeaboss](https://github.com/ravenac95/sudolikeaboss), by default it is required to set the value of the
+`website` field for each login item with the value of `sudolikeaboss://local`. To override this behavior and provide
+customized filtering, see configuration options `@1password-enabled-url-filter` and `@1password-items-jq-filter` below.
 
```

Comment:
I would add a "Security" section to the README explaining that we're saving some data (and what data) from the vaults to the disk.